js/renovation/ui/editors/__tests__/check_box.test.tsx

```diff
@@ -199,6 +199,47 @@ describe('CheckBox', () => {
     describe('Effects', () => {
       afterEach(clearEventHandlers);
 
+      describe('updateIconFontSize', () => {
+        it('should set icon font size on init', () => {
+          const checkBox = new CheckBox({ iconHeight: 22, iconWidth: 22 });
+          checkBox.iconRef = { current: { style: {} } } as any;
+          const icon = checkBox.iconRef.current;
+
+          checkBox.updateIconFontSize();
+
+          expect(icon?.style.fontSize).toEqual('16px');
+        });
+
+        it('should change icon font size after runtime changes in "iconWidth"/"iconHeight" options', () => {
```

Comment:
if we don't change `props.iconWidth`, no change will be in test result. Let's write seperate tests for iconWidth/iconHeight

Reply:
created 2 tests: 1 with decreasing width (font size should change), 2 with increasing height (font size should not change)